common/src/cellular_pktio.c

```diff
@@ -1073,17 +1073,6 @@ static void _PktioInitProcessReadThreadStatus( CellularContext_t * pContext )
 
 /*-----------------------------------------------------------*/
 
-void _Cellular_PktioSetShutdownCallback( CellularContext_t * pContext,
```

Comment:
There is a function declaration in ./include/private/cellular_pktio_internal.h.

```
/**
 * @brief Packet IO shutdown callback set function.
 *
 * This function set the shutdown callback set function.
 * Once the packet IO thread is shutdown, the packet IO thread will call this function.
 *
 * @param[in] pContext The opaque cellular context pointer created by Cellular_Init.
 * @param[in] shutdownCb The shutdown callback function. Suggest we also remove it.
 */
void _Cellular_PktioSetShutdownCallback( CellularContext_t * pContext,
                                         _pPktioShutdownCallback_t shutdownCb );
```

Reply:
Yes, we need removed it in header file. The original APIs didn't let user changes the callback function, so I change it to static function.
```
_Cellular_PktioSetShutdownCallback( pContext, _shutdownCallback );
-->static void _Cellular_SetShutdownCallback( CellularContext_t * pContext,
                                           _pPktioShutdownCallback_t shutdownCb )
```